mysite/templates/start.html

```diff
@@ -7,21 +7,22 @@
 
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-    <link rel="stylesheet" href="{{ url_for('static', filename='css/style.css') }}">
```

Comment:
Hi Tom, why did you replace the url_for functionality with absolute path to the style and favicon? I think the url_for will be more portable and in line with flask file systems. Can this be reverted?